Makefile

```diff
@@ -9,16 +9,9 @@ BUILDDIR      	 	 = docs/_build
 .PHONY: clean html
 
 dev: full-environment.yml
-	$(PY_ENV_MANAGER) create -f full-environment.yml -y
-	$(PY_ENV_MANAGER) -n hydromt run pip install .
-	$(PY_ENV_MANAGER) -n hydromt run pre-commit install
-
-env:
-	@# the subst is to make sure the is always exactly one "" around OPT_DEPS so people can
-	@# specify it both as OPT_DEPS=extra,io and OPT_DEPS="extra,io"
-	python3 make_env.py "$(subst ",,$(OPT_DEPS))"
-	$(PY_ENV_MANAGER) create -f environment.yml -y
-	$(PY_ENV_MANAGER) -n hydromt run pip install .
+	$(PY_ENV_MANAGER) env create -f full-environment.yml
+	$(PY_ENV_MANAGER) run -n hydromt-full pip install -e .
+	$(PY_ENV_MANAGER) run -n hydromt-full pre-commit install
 
```

Comment:
I assume this was deleted because of the changes i requested in make_env.py? Does it make sense to keep it but for single dependency groups or flavours only? If we delete it we should also remove a mention of it in the (dev/dev_install.rst) and the OPT_DEPS global argument above.

Reply:
good point, I originally removed it just because it felt a little redundant with the flavours but on reading your comment I think it is still a good one to keep so I"ll add it back in. The one downside with it is that the environment will have a slightly weird name like `hydromt-extra,io` but I'm not sure that can be helped.